microsoftgraph/microsoft-graph-explorer-v4
# Fix infinite spinning when login fails

## Overview

Fixes #487 

### Demo

![image](https://user-images.githubusercontent.com/58787602/80581397-c4dd3880-8a15-11ea-9dbc-e981757eaf5c.png)


## Testing Instructions

* Click on sign in
* Once the pop up loads, dismiss it
* Notice the spinning stops and the a status message is displayed
